pages/[id]/spocos.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import Page from '../../components/page';
 import { getHackathonPaths, getHackathons } from '../../utility/firebase';
+import SpocosPage from '../../components/spocos';
 
 export default ({ id, hackathons }) => (
   <Page currentPath={id} hackathons={hackathons}>
-    Sponsors
+    <SpocosPage name={id} />
```

Comment:
if this is really an id it might be better to pass this in as an id instead of a name? 
```suggestion
    <SpocosPage id={id} />
```

Reply:
i use the name prop for database pathing purpose, unless id is used for something different

Reply:
In that case I think we should use id. Technically `getSponsors` gets the sponsor data using an id. And we should also update `getSponsors` to take an `id` instead of a `website`